App monitoring + Open app + App authentication (#1067)
akeneo/pim-helpcenter: content/md/connectivity/manage-your-connections.md

## The `Source connection` flow type

> Choose this option for your connection whenever it represents a data flow entering the PIM. For example, your data flow is mainly creating or updating PIM data, such as products, product models, assets, reference entities,...

Reviewer: ```suggestion
Choose this option for your connection whenever it represents a data flow entering the PIM. For example, your data flow mainly creates or updates PIM data, such as products, product models, assets, reference entities,...
```

::: info

> With this flow type, you can connect ERP, DAM, MDM,...

Reviewer: ```suggestion
With this flow type, you can connect your ERP, DAM, MDM, etc.
```

## The `Destination connection` flow type

> Choose this option for your connection whenever it represents a data flow that is mainly extracting product information from your PIM.

Reviewer: ```suggestion
Choose this option for your connection whenever it represents a data flow that mainly extracts product information from your PIM.
```

> # Enable the tracking

Reviewer: ```suggestion
# Enable the monitoring
```

> For each connection, you can choose whether you want to track it in [Data flows](connection-dashboard.html).  

Reviewer: ```suggestion
For each connection, you can choose whether you want to monitor it in [Data flows](connection-dashboard.html).  
```

> To do so, check `Track in Data flows dashboard`, in the connection setting screen of the connection you want to track.

Reviewer: ```suggestion
To do so, check `Track in Data flows dashboard`, in the connection settings screen of the connection you want to track.
```

::: info

> You can deactivate the tracking whenever you want.  

Reviewer: ```suggestion
You can disable the monitoring whenever you want.  
```

> We will stop tracking the connection immediately and you won't be able to select it in the filters of the Data flows dashboard. However, note that all the data collected up to that point for this connection will still appear in some KPIs.

Reviewer: ```suggestion
We will stop monitoring the connection immediately and you won't be able to select it in the filters of the Data flows dashboard. However, note that all the data collected up to that point for this connection will still appear in some KPIs.
```
